lib/abide_dev_utils/cli/xccdf.rb

```diff
@@ -32,7 +32,7 @@ def initialize
         options.on('-d [DIR]', '--files-output-directory [DIR]', 'Directory to save files data/mappings by default') { |d| @data[:dir] = d }
         options.on('-q', '--quiet', 'Show no output in the terminal') { @data[:quiet] = true }
         options.on('-p [PREFIX]', '--parent-key-prefix [PREFIX]', 'A prefix to append to the parent key') do |p|
-          @data[:parent_key_prefix] = p
+          @data[:prefix] = p
```

Comment:
This just gets reassigned to `@data[:parent_key_prefix]` on line 46, why the change?

Reply:
Was keeping with the trend of the stuff in [] matching @data[:]